locales/en/translation.json

```diff
@@ -75,7 +76,13 @@
     "module-lock-info": "This module is license-protected and requires that you purchase an unlock key from the content owner!",
     "unlock-key": "Unlock key",
     "enter-unlock-key": "{{moduleId}} is license protected. Please enter the unlock key!",
-    "unlock-failed": "Unlock failed! Please enter the correct unlock key."
+    "unlock-failed": "Unlock failed! Please enter the correct unlock key.",
+    "app-system-languages": "Used and System Languages",
+    "bible-languages": "Bible Languages",
+    "most-speaking-languages": "Most Spoken Languages",
+    "historical-languages": "Historical Languages",
+    "other-languages": "Other Languages",
+    "search-result": "Matches Found"
```

Comment:
Question: for locale strings that changed but have the same key (`module-assistant.pick-languages-from-repos`, `module-assistant.repo-selection-info-text`, etc.) should I just put new English text for all locales or is it better to use new keys instead?

Reply:
I would suggest to copy & paste the new English text.

Reply:
I went through all the texts for module installation with my father in law. He is semi-technical. I got more updates than I initially expected. But in the end it turned out to be more clear for the common user. I realized that it's important to avoid term "module" completely. I added quite a few new keys to reflect the updated text.